static/js/rendered_markdown.js

```diff
@@ -218,7 +218,7 @@ export const update_elements = (content) => {
                 } else {
                     view_in_playground_button.attr("aria-haspopup", "true");
                 }
-                view_in_playground_button.attr("title", title);
+                view_in_playground_button.attr("data-tippy-content", title);
```

Comment:
This change is good, but I notice a vertical alignment mismatch between the two adjacent buttons that makes the tooltips look a bit off.  Probably not worth spending much time fixing since I think we'll be changing the icon, but if it's a couple pixels of incorrect vertical alignment on the icons themselves, it's probably worth fixing that.

@sumanthvrao FYI.

Reply:
https://github.com/zulip/zulip/pull/18042